config/lava/chromeos/tast.jinja2

```diff
@@ -0,0 +1,33 @@
+- test:
+      timeout:
+        minutes: 45
+      docker:
+        image: {{ docker_image }}
+        volumes:
+          - /tmp/octopus:/home/cros-tast/octopus
+        wait:
+          device: true
+      results:
+          location: /home/cros-tast/lava
+      definitions:
+      - repository:
+          metadata:
+            format: Lava-Test Test Definition 1.0
+            name: docker-ping
+          run:
+            steps:
+              - 'cd /home/cros-tast'
+{%- if cros_tast_tarball %}
+              - 'curl -s {{cros_tast_tarball}} | tar xjf -'
+{% endif %}
```

Comment:
Is it not mandatory to have a Tast tarball to run this job?

Reply:
We might have either rootfs_url in case of ChromiumOS opensource build (then we fetch relevant tast and remote_test_runner), or we might have cros_tast_tarball, tast tarball supplied locally (as we had on proprietary build tests)

Reply:
Done

Reply:
If it's mandatory then we should remove the `{%- if` statement.